### PR TITLE
added docker-build module and fixed rosbag readme

### PR DESCRIPTION
**Pull request by srinivasreddych**
*Issue #, if available:*

Adds a Docker build module and scans the container image built.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: manifests/example-prod/ide-modules.yaml [hunk 1]
```diff
@@ -21,56 +21,56 @@ parameters:
...
-    valueFrom:
-      moduleMetadata:
-        group: optionals
-        name: datalake-buckets
-        key: ArtifactsBucketName
-  - name: sso-username
-    value: replace-me
-  - name: emr-eks-namespace
-    value: emr-studio
+# ---
```
Comment (chamcca) [inline manifests/example-prod/ide-modules.yaml:24]: we probably need to drop this beta module from example-prod completely, not just comment it.

**Review by dgraeber — COMMENTED** [review]
Gonna need a workflow entry to to do validation checks:

.github/workflows/main.yml

File: manifests/example-prod/ide-modules.yaml (hunk 1 above)
Comment (srinivasreddych) [reply manifests/example-prod/ide-modules.yaml:24]: Thanks for catching it. As discussed offline earlier, i have removed the entries of emrstudio in the release/0.1.0 but i missed addressing them in main.

**Comment by srinivasreddych** [comment]
> Gonna need a workflow entry to to do validation checks:
> 
> .github/workflows/main.yml

I haven't noticed a validation check for `example-dags` module, and assumed we are not doing for `examples`. Should i add validation checks for examples as well?

**Pull request merged**